crates/aptos-runtimes/src/lib.rs

```diff
@@ -24,6 +24,8 @@ pub fn spawn_named_runtime_with_start_hook<F>(
 where
     F: Fn() + Send + Sync + 'static,
 {
+    const MAX_BLOCKING_THREADS: usize = 64;
```

Comment:
nit: can we put this at the top by the other const? 😄

Reply:
Can we make this configurable in the node config?

Reply:
@JoshLind I prefer limiting it to the smallest scope necessary.. we can always move it if needed.

@banool: too many places to change.. don't think it's worth it at this point. (too lazy)